src/main/java/com/redhat/ecosystemappeng/morpheus/rest/FeedbackResource.java

```diff
@@ -0,0 +1,62 @@
+package com.redhat.ecosystemappeng.morpheus.rest;
+
+import com.redhat.ecosystemappeng.morpheus.model.FeedbackDto;
+import com.redhat.ecosystemappeng.morpheus.service.FeedbackService;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.jboss.logging.Logger;
+
+@Path("/feedback")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class FeedbackResource {
+    private static final Logger LOGGER = Logger.getLogger(FeedbackResource.class);
+
+    @Inject
+    FeedbackService feedbackService;
+    @POST
+    public Response forwardToFlask(FeedbackDto dto) {
```

Comment:
@olavtar IMO it's better to rename it to the service name instead of the pythonic  HTTP framework
```suggestion
    public Response forwardToUserFeedbackService(FeedbackDto dto) {
```

Reply:
Done